schema/func_yaml-schema.json

```diff
@@ -1,420 +0,0 @@
-{
```

Comment:
Why is this file deleted?

Reply:
Sorry, I accidentally deleted that file...

I hope everything is okay now!

Reply:
One clarification: I built the project following the [CONTRIBUTING.md](https://github.com/knative/func/blob/main/docs/CONTRIBUTING.md) guide.
In particular, I ran `make build` -> `make clean` -> `make test`.

Is it correct that the `make clean` command deletes the _schema/func_yaml-schema.json_ file?
https://github.com/knative/func/blob/5bb373ab2ca4169437daf011f087b42c47960480/Makefile#L104C1-L109C20

Reply:
I am not sure why it's deleting that file.